src/utils.ts

```diff
@@ -75,6 +77,19 @@ class Utils {
 		return Promise.resolve(template);
 	}
 
+	private static _getConfigWithRandomMailConfig(config:any): any {
+		if (Array.isArray(config.mail) ){
+			config.hasMultipleAccounts = true;
+			config.mail = this._getRandomElementfromList(config.mail);
+		}
+		return config;
+	}
+
+	private static  _getRandomElementfromList (array: Array<any>) {
+		const randPos:number = Math.floor((Math.random()*array.length));
```

Comment:
the random seed should be stored globally to take care the value is uniformly distributed. instead, why not take every time the next config ?